pkg/registry/falcon_registry/node.go

```diff
@@ -15,7 +15,7 @@ func (reg *FalconRegistry) LastNodeTag(ctx context.Context, versionRequested *st
 
 	return lastTag(ctx, systemContext, reg.imageUriNode(), func(tag string) bool {
 		return (tag[0] >= '0' && tag[0] <= '9' &&
-			strings.Contains(tag, ".falcon-linux.x86_64") &&
```

Comment:
We did not have a similar filter for `AARCH64` ?

Reply:
We do/did not.